driver-core/src/main/com/mongodb/client/model/Accumulators.java

```diff
@@ -173,10 +183,123 @@
      * @since 3.2
      */
     public static <TExpression> BsonField stdDevSamp(final String fieldName, final TExpression expression) {
-        return accumulator("$stdDevSamp", fieldName, expression);
+        return accumulatorOperator("$stdDevSamp", fieldName, expression);
+    }
+
+    /**
+     * Creates an $accumulator pipeline stage
+     *
+     * @param initFunction         a function used to initialize the state
+     * @param accumulateFunction   a function used to accumulate documents
+     * @param mergeFunction        a function used to merge two internal states, e.g. accumulated on different shards or threads. It
+     *                             returns the resulting state of the accumulator.
+     * @return the $accumulator pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/accumulator/ $accumulator
+     * @mongodb.server.release 4.4
+     * @since 4.1
+     */
+    public static Bson accumulator(final String initFunction, final String accumulateFunction, final String mergeFunction) {
```

Comment:
The methods in this class should all be consistent with each other.  That is, they should all take a `fieldName` as the first parameter, and they should all return a `BsonField`.

Reply:
I have altered the methods so that the first parameter is `fieldName` and they return `BsonField`.